### ISSUE TITLE
Length Histograms in texts columns are being wrongly plotted

### ISSUE THREAD

**Issue by leopascual** [pandas-profiling/pandas-profiling#642]
**Describe the bug**

Assuming all text columns length frequency histograms are referring to the length on values found on the text columns, I noticed several columns that have only two possible values for lengths col1(null, 13);  col2 (null, 8) but the profiling report calculates mins, mean, median not found in the table data. 

**To Reproduce**
Use a single column with multiple values of unique length combine with null values: should only exist one bin for the total frequency of values in DF. 

_Data:_
<class 'pandas.core.frame.DataFrame'>
RangeIndex: 215861 entries, 0 to 215860
Data columns (total 20 columns):
 ### No  Column              Non-Null Count   Dtype  
 0   gr_member_id        215861 non-null  int64  
 1   codParticipante_id  215861 non-null  int64  
 2   Nat_id              161176 non-null  object 
 3   PID                 215859 non-null  float64
 4   Participant_Number  165598 non-null  object 
 5   National_ID         161176 non-null  object 
 6   First_Name          65002 non-null   object 
 7   Middle_Name         54553 non-null   object 
 8   Last_Name           64867 non-null   object 
 9   Second_Last_Name    57914 non-null   object 
 10  Complete_Name       215846 non-null  object 
 11  Date_Of_Birth       165750 non-null  object 
 12  Gender              165746 non-null  object 
 13  Marital_Status      165705 non-null  object 
 14  Country_Of_Origin   163872 non-null  object 
 15  Mobile              65701 non-null   object 
 16  Address             165596 non-null  object 
 17  isBiometric         65002 non-null   object 
 18  isPoliticalExp      65002 non-null   object 
 19  professionEs        64996 non-null   object 
dtypes: float64(1), int64(2), object(17)
memory usage: 32.9+ MB

_Code_:
```from sqlalchemy import create_engine
import pymysql
import pandas
import pandas_profiling 

db_connection_str = 'mysql+pymysql://root:@localhost/cliente_unico'
db_connection = create_engine(db_connection_str)
profile_member = master_member.profile_report(title="Member Profiling Report")
#profile_affiliates.set_variable('variables.descriptions', desc_affil)
profile_member.to_notebook_iframe()
```

**Version information:**

Everything has recently installed, downgrade from python 3.9 to 3.7 because incompatibility with pandas-profiling==2.9.0
* _Python version_: 3.7.9
* _Environment_:  Jupyter Notebook Local
* _`pip`_: (jupy_venv) PS C:\python\jupyter> pip freeze
appdirs==1.4.4
argon2-cffi==20.1.0
async-generator==1.10
attrs==20.3.0
autoflake==1.4
backcall==0.2.0
black==20.8b1
bleach==3.2.1
certifi==2020.11.8
cffi==1.14.4
chardet==3.0.4
clean-ipynb @ git+https://github.co
click==7.1.2
colorama==0.4.4
confuse==1.4.0
cycler==0.10.0
decorator==4.4.2
defusedxml==0.6.0
entrypoints==0.3
htmlmin==0.1.12
idna==2.10
ImageHash==4.2.0
importlib-metadata==3.1.0
ipykernel==5.3.4
ipython==7.19.0
ipython-genutils==0.2.0
ipywidgets==7.5.1
isort==5.6.4
jedi==0.17.2
Jinja2==2.11.2
joblib==0.17.0
jsonschema==3.2.0
jupyter-client==6.1.7
jupyter-core==4.7.0
jupyterlab-pygments==0.1.2
kiwisolver==1.3.1
llvmlite==0.34.0
MarkupSafe==1.1.1
matplotlib==3.3.3
missingno==0.4.2
mistune==0.8.4
mypy-extensions==0.4.3
nbclient==0.5.1
nbconvert==6.0.7
nbformat==5.0.8
nest-asyncio==1.4.3
networkx==2.5
notebook==6.1.5
numba==0.51.2
numpy==1.19.4
packaging==20.7
pandas==1.1.4
pandas-profiling==2.9.0
pandocfilters==1.4.3
parso==0.8.0
pathspec==0.8.1
phik==0.10.0
pickleshare==0.7.5
Pillow==8.0.1
prometheus-client==0.9.0
prompt-toolkit==3.0.8
pycparser==2.20
pyflakes==2.2.0
Pygments==2.7.2
PyMySQL==0.10.1
pyparsing==2.4.7
pyrsistent==0.17.3
python-dateutil==2.8.1
pytz==2020.4
PyWavelets==1.1.1
pywin32==300
pywinpty==0.5.7
PyYAML==5.3.1
pyzmq==20.0.0
regex==2020.11.13
requests==2.25.0
scipy==1.5.4
seaborn==0.11.0
Send2Trash==1.5.0
six==1.15.0
SQLAlchemy==1.3.20
tangled-up-in-unicode==0.0.6
terminado==0.9.1
testpath==0.4.4
toml==0.10.2
tornado==6.1
tqdm==4.54.0
traitlets==5.0.5
typed-ast==1.4.1
typing-extensions==3.7.4.3
urllib3==1.26.2
visions==0.5.0
wcwidth==0.2.5
webencodings==0.5.1
widgetsnbextension==3.5.1
zipp==3.4.0

**Additional context**
![image](https://user-images.githubusercontent.com/43121349/101288928-c4ccc480-37be-11eb-8458-ae128a0bce9e.png)
<img width="1280" alt="Screen Shot 2020-12-06 at 11 23 35 AM" src="https://user-images.githubusercontent.com/43121349/101288938-ce562c80-37be-11eb-9a01-521dc4208ce4.png">

**Comment by sbrugman** [comment]
Thanks for reporting this @leopascual in such detail. Seems like a duplicate of #613, for which a fix is started in #619. 

> Everything has recently installed, downgrade from python 3.9 to 3.7 because incompatibility with pandas-profiling==2.9.0
I'd be interested in knowing where the incompatibility comes from, we might be able to resolve this.

**Comment by leopascual** [comment]
Hi @sbrugman, 
I have created a virtualenv with 3.9.0, upgrade pip, install pandas; and then after, when I try to install pandas-profiling an error during installation is occurring.  Attached complete detail. 
Building wheels for collected packages: numba, llvmlite
  Building wheel for numba (setup.py) ... error
  ERROR: Command errored out with exit status 1:

[error_installing_pp.txt](https://github.com/pandas-profiling/pandas-profiling/files/5655650/error_installing_pp.txt)

**Comment by sbrugman** [comment]
@leopascual  I've just released v2.10.1 that should resolve your issue!

_Want me to continue to support this package? Please consider [sponsoring](https://github.com/sponsors/sbrugman)_
